### PR TITLE
[Merged by Bors] - chore: fix typo `Continous`-> `Continuous`

### PR DESCRIPTION
**Pull request by sgouezel**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by YaelDillies — APPROVED** [review]
Glad I'm not the only one with dubous spelling 🙈 

maintainer merge

File: Mathlib/MeasureTheory/Measure/FiniteMeasure.lean [hunk 1]
```diff
@@ -645,7 +645,7 @@ theorem tendsto_iff_forall_integral_tendsto {γ : Type*} {F : Filter γ} {μs :
   simp_rw [aux, BoundedContinuousFunction.toReal_lintegral_coe_eq_integral] at tends_pos tends_neg
   exact Tendsto.sub tends_pos tends_neg
 
-lemma continuous_integral_boundedContinousFunction
+lemma continuous_integral_boundedContinuousFunction
```
Comment (urkud) [inline Mathlib/MeasureTheory/Measure/FiniteMeasure.lean:648]: You added deprecated aliases for other renamed lemmas but not for this one. Is it intentional?
bors d+

Comment (YaelDillies) [reply Mathlib/MeasureTheory/Measure/FiniteMeasure.lean:648]: This one xas added a few day ago by myself.< It's ony usd in PFR

Comment (sgouezel) [reply Mathlib/MeasureTheory/Measure/FiniteMeasure.lean:648]: Yes, it's so recent that it's not worth deprecating.

**Pull request closed without merging**